ASSETS.md

```diff
@@ -259,6 +259,88 @@ but it can be triggered through URL parameters.
 The Kibana Space API can be found [here](https://www.elastic.co/guide/en/kibana/master/spaces-api.html). Kibana Spaces
 are not saved objects and have their own API.
 
+### Input
+
+* Asset Path: `input/{input-name}/{package-structure}`
+
+All inputs are defined inside the `input` directory. An example here is the `access` input of the `nginx` package.
+Inside each input, the same structure is repeated which is defined for the overall package. In general ingest pipelines
+and fields definitions are only expected inside input. 
+
+**manifest.yml**
+
+Each input must contain a manifest.yml. It contains all information about the input and how to configure it.
+
+```
+# Needs to describe the type of this input. Currently either metric or log
+type: metric
+
+# Each input can be in its own release status
+release: beta
```

Comment:
@hbharding Some inputs can also be in beta. We probably need some design to indicate this on the create data source page where inputs can be enabled / disabled.